### DISCUSSION TITLE
Search Filters / Ui

### DISCUSSION THREAD

**Discussion by hunterallen** [community/community#3936] [Mobile] [answered]
Any way to use search filters from iOS app? From the browser, it's easy to choose which language you want to filter for, but if it's possible from the app, it's not intuitive.

**Comment by Iron-Ham** [comment, marked as answer]
Hi @hunterallen. At the moment, we do support using [GitHub Search Syntax](https://docs.github.com/en/github/searching-for-information-on-github/understanding-the-search-syntax) in all of our search boxes. 

But stay tuned, we're thinking on a more robust solution for this in the future.

**Reply by mediter** [reply]
3 years later, still no change in this regard.
